qt/applications/workbench/workbench/plugins/memorywidget/memorypresenter.py

```diff
@@ -0,0 +1,29 @@
+from qtpy.QtCore import QTimer
+
+from ..memorywidget.memoryinfo import get_memory_info
+
+
+class MemoryPresenter(object):
```

Comment:
Can we add method docs to everything so we can signal to future developers what the intent is? An example of docs in Python is in (algorithm_factory_observer)[https://github.com/mantidproject/mantid/blob/master/qt/python/mantidqt/widgets/algorithmselector/algorithm_factory_observer.py]